src/pt/manhastro/src/eu/kanade/tachiyomi/extension/pt/manhastro/Manhastro.kt

```diff
@@ -26,6 +26,8 @@ class Manhastro : Madara(
 
     override val mangaDetailsSelectorTitle = "div.summary_content h2"
 
+    override val mangaDetailsSelectorStatus = "div.post-status .post-content_item:nth-child(2) > .summary-content"
```

Comment:
Could change position

```suggestion
    override val mangaDetailsSelectorStatus = "div.summary-heading:contains(Status) + div.summary-content"
```

Reply:
Cool, I was just using standard(spec) CSS selectors.